lib/cli.js

```diff
@@ -236,6 +238,11 @@ const meowOptions = {
       --allow-empty-input, --aei
 
         When glob pattern matches no files, the process will exit without throwing an error.
+	 
+	  --globby-options, --go
+
+		CLI option to traverse dotfiles i.e Hidden folders files will not be ignored(e.g. stylelint --fix '**/*.css)
+	
```

Comment:
[suggest] Let's remove extra empty lines and convert TAB to SPACEs.

```suggestion

      --globby-options, --go

        CLI option to traverse dotfiles i.e Hidden folders files will not be ignored(e.g. stylelint --fix '**/*.css)	
```